Feature/crawling/hansol

Kain7f1 opened the pull request:
크롤링 파일들을 추가하고, 수정했습니다
requirements.txt도 업데이트 함

Kain7f1 committed:
Make requirements.txt

requirements.txt 를 만들었습니다

Kain7f1 committed:
Add utility_module.py, dcinside_url_crawler.py, test_dcinside_url.py

[add]
utility_module.py, dcinside_url_crawler.py, test_dcinside_url.py

[modify]
requirements.txt

heexo commented on crawling_url/src/dcinside_url_crawler.py:
```diff
@@ -0,0 +1,68 @@
+def get_url_dc_toss(keyword, start_date, end_date):
+    media = "dcinside_toss"  # 토스 갤러리 media column
+    header = {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36"}
+    folder_path = f"./{media}/url"  # 저장할 폴더 경로 설정
```
폴더 계층 재정립 필요 
- crawling/{media}/url
- crawling/{media}/content 
-

Kain7f1 replied:
crawling_url과 crawling_content를 합칠게요

heexo reviewed (changes requested):
수정이 필요합니다 내일해주세요

Kain7f1 committed:
Change Folder Structure

[add]
utility_module.py, dcinside_url_crawler.py, test_dcinside_url.py

[modify]
requirements.txt
